breakout_2/breakout_2_answers.Rmd

```diff
@@ -0,0 +1,78 @@
+---
+title: "Breakout 2: Combining Data Frames"
+subtitle: "PHUSE US Connect 2021: Introduction to R for the Statistical Programmer"
+author: "Atorus Research"
+output:
+  rmarkdown::html_document:
+    theme: lumen
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(tidyverse)
+library(haven)
+library(knitr)
+```
+
+## Intro
+
+We're going to work with some data from the DM dataset.
+
+```{r load_data}
+# Use haven to read the raw data straight off of GitHub!
+raw_dm <- read_xpt(url("https://github.com/phuse-org/TestDataFactory/raw/main/Updated/TDF_SDTM/dm.xpt"))
+raw_ae <- read_xpt(url("https://github.com/phuse-org/TestDataFactory/raw/main/Updated/TDF_SDTM/ae.xpt"))
+```
+
+We'll pick up where we left off from the last exercise with the subsets on the data.
+
+```{r subset}
+dm <- raw_dm %>% 
+  select(USUBJID, STUDYID, SITEID, SUBJID, ARM, AGE, RFSTDTC) %>% 
+  filter(RFSTDTC != "")
+
+total_dm <- dm %>% 
+  mutate(ARM = 'Total')
```

Comment:
I would add a note that they should create this dataset

Reply:
@kodesiba you combined the cells - but add a note to the user to make sure they run this chunk before continuing on with the exercise.